test/integration/agent/openflow_test.go

```diff
@@ -266,7 +266,7 @@ func testInstallTunnelFlows(t *testing.T, config *testConfig) {
 }
 
 func testInstallServiceFlows(t *testing.T, config *testConfig) {
-	err := c.InstallClusterServiceFlows()
+	err := c.InstallClusterServiceFlows(true, true)
```

Comment:
out of curiosity, with IPv6 introduced, why there's no change to the expected output of openflow in the test?

Reply:
Service cluster IP still NATs the pod IP, even with IPv6. Right?

Reply:
Service cluster IP still NATs the pod IP, even with IPv6. Right?

Reply:
Where the test does validate the IPs... I didn't change the original tests functionality, but modified the existing ones to cover IPv6 when the IPs were validated.